plugins/psgi/psgi_plugin.c

```diff
@@ -3,11 +3,7 @@
 extern char **environ;
 extern struct uwsgi_server uwsgi;
 
-#ifdef __APPLE__
```

Comment:
Won't this break __APPLE__ though?

Reply:
I don't think so? this is made so it matches what `__APPLE__` did

Reply:
right, let's merge this